src/components/ListItem.jsx

```diff
@@ -1,5 +1,45 @@
+import { useEffect, useState } from 'react';
+import { updateItem } from '../api/firebase';
+import { ONE_DAY_IN_MILLISECONDS } from '../utils';
 import './ListItem.css';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ listPath, item, name }) {
```

Comment:
This is a nitpick and not exclusive to your code but includes the other issue worked on this week; how will we deal with the merge conflict that I am forseeing on the horizon? Right now this code renders whether there are any items in the list or not, but Iryna and I edited the List component to render separate messages and items if there are items in the list or not, so I feel that this entire functionality of the checkboxes should also only render with its prompts if there are items in the list. However, that issue will only be able to be handled once the code is brought together. Thoughts?

Reply:
Hey Paige! The issue I'm seeing right now is that the `Mark your purchases...` prompt appears whether the list has items or not. We can potentially fix this by conditionally rendering the prompt the same way as the other prompts. Also a good opportunity to revise our prompt and make it a little clearer.

```javascript
{data.length > 0 && (
   <div>
      <p>Mark your purchases, they will automatically uncheck after 24hrs.</p>
      <p>You can uncheck if you no longer want to make the purchase.</p>
   </div>
)}			
```

If we revise our code and add your changes first, I don't think there should be any issues adding this together with the other conditional prompts. (@rachelspencer do you agree?) 

Let us know what you think!

Reply:
Yes sounds like a plan!